interface/new/new_comprehensive_save.php

```diff
@@ -28,11 +28,9 @@
 // other interfaces can still read the data during this lock, however
 // sqlStatement("lock tables patient_data read");
 
-$result = sqlQuery("SELECT MAX(pid)+1 AS pid FROM patient_data");
-
+$result = sqlQuery("SELECT id AS pid FROM patient_data ORDER BY id DESC LIMIT 1");
```

Comment:
Also have changes in 2 unrelated files here:
interface/documents/upload.php
templates/documents/general_upload.html